src/relay.js

```diff
@@ -0,0 +1,32 @@
+import {fromGlobalId, connectionFromArray, nodeDefinitions} from 'graphql-relay';
+
+export function idFetcher(sequelize) {
+  return globalId => {
+    let {type, id} = fromGlobalId(globalId);
+    const models = Object.keys(sequelize.models);
+    type = type.toLowerCase();
```

Comment:
The reason we need it is because of the possibility of defining types like what we did:

```
id: globalIdField('Task'),
```

Just to make sure that in the event someone types the models with upper case name its forced to lower case to test for equality in the models object